sensors/Makefile

```diff
@@ -4,26 +4,29 @@
 # Copyright 2022 Phoenix Systems
 #
 
+LOCAL_PATH := $(call my-dir)
+
 # Sensors Server API Library
 NAME := libsensors
-include $(static-lib.mk)
-
-# Sensors SPI interface
-NAME := libsensors-spi
-LOCAL_HEADERS := sensors-spi.h
-LOCAL_HEADERS_DIR := nothing  # do not reinstall libsensors.h
-LOCAL_SRCS := libsensors-spi.c
+SRCS := $(wildcard $(LOCAL_PATH)common/*/*.c)
 DEPS := libzynq7000-gpio-msg libspi-msg
 include $(static-lib.mk)
 
-# Sensors Server
-SENSORS_ALL := imu/lsm9dsxx imu/mpu6000 mag/lsm9dsxx mag/lis2mdl baro/lps25xx baro/ms5611 gps/pa6h gps/ubx
-SENSORS_SIM := imu/imu_sim.c mag/mag_sim.c baro/baro_sim.c gps/gps_sim.c gps/gps_sim.c
-SENSORS_LOCAL := $(SENSORS_SIM) $(addsuffix .c, $(filter $(SENSORS_ALL), $(DEVICE_SENSORS)))
+# Sensors application
+SENSORHUB_COMMON_DEPS := libzynq7000-gpio-msg libspi-msg libsensors
+SENSORHUB_DRIVERS_ALL :=
+
+ALL_SENSOR_MAKES := $(shell find . -mindepth 2 -name sensors.mk -not -path '*/.*')
+include $(ALL_SENSOR_MAKES)
+
+SENSORHUB_DRIVERS_ENABLED ?= baro_sim imu_sim mag_sim gps_sim
```

Comment:
hmm, maybe add comment that this is the default sensors list and the intent is to change it externally on a per-project basis?

Reply:
Added!